SDL/Kernels.cu

```diff
@@ -877,19 +878,149 @@ __device__ int inline checkHitsT5(unsigned int ix, unsigned int jx,struct SDL::m
     }
     return nMatched;
 }
+__device__ int inline checkHitspT5(unsigned int ix, unsigned int jx,struct SDL::pixelQuintuplets& pixelQuintupletsInGPU)
+{
+    unsigned int hits1[14];// = {-1,-1,-1,-1,-1,-1,-1,-1,-1,-1};
+    unsigned int hits2[14];// = {-1,-1,-1,-1,-1,-1,-1,-1,-1,-1};
+
+
+    hits1[0] = pixelQuintupletsInGPU.hitIndices[14*ix];
```

Comment:
Instead of creating new memory locations, we can initialize a pointer to the first hit and then index starting from there. I'll make this change in my PR

Reply:
On second thought, looks like changing this to direct accessing from global memory using pointers increases the timing of pT5 by a full 8ms for the first event (50%). So I won't make this change